src/applications/vaos/actions/newAppointment.js

```diff
@@ -101,6 +109,45 @@ export function openFacilityPage(page, uiSchema, schema) {
   };
 }
 
+export function updateFacilityPageData(page, uiSchema, data) {
```

Comment:
This is called when any data changes, which means either the system or facility. It dispatches the normal form data update action, then checks to see if we need to fetch more facilities.